src/components/modals/modals.jsx

```diff
@@ -5,22 +5,35 @@ import {
 } from 'reactstrap';
 
 export const ModalWindow = ({
-  isOpen, toggleInModal, header, toggleInHeader, children, buttonLeft,
-  buttonRight, buttonLeftHandleClick, buttonRightHandleClick,
+  isOpen, toggleInModal, header, toggleInHeader, buttonLeft,
+  buttonRight, buttonLeftHandleClick, buttonRightHandleClick, children, isButtonRightDisable,
 }) => (
   <Modal isOpen={isOpen} toggle={toggleInModal}>
     <ModalHeader toggle={toggleInHeader}>{header}</ModalHeader>
     <ModalBody>
       {children}
     </ModalBody>
     <ModalFooter>
-      <Button color="primary" onClick={buttonLeftHandleClick}>{buttonLeft}</Button>
-      <Button color="secondary" onClick={buttonRightHandleClick}>{buttonRight}</Button>
+      <Button
+        color="primary"
+        type="submit"
+        onClick={buttonLeftHandleClick}
```

Comment:
it's not a good idea to distinguish buttons by left and right. You can say that one of them is primary, another one is secondary